cmd/podman/play/kube.go

```diff
@@ -75,6 +77,10 @@ func init() {
 	flags.StringVar(&kubeOptions.Authfile, authfileFlagName, auth.GetDefaultAuthFile(), "Path of the authentication file. Use REGISTRY_AUTH_FILE environment variable to override")
 	_ = kubeCmd.RegisterFlagCompletionFunc(authfileFlagName, completion.AutocompleteDefault)
 
+	staticIPFlagName := "ip"
+	flags.StringVar(&kubeOptions.StaticIPCLI, staticIPFlagName, "", "Static IP address to assign to this pod")
+	_ = kubeCmd.RegisterFlagCompletionFunc(staticIPFlagName, completion.AutocompleteDefault)
+
```

Comment:
You can just use an IP flag. No need for the extra parsing logic
```suggestion
	staticIPFlagName := "ip"
	flags.IPVar(&kubeOptions.StaticIP, staticIPFlagName, nil, "Static IP address to assign to this pod")
	_ = kubeCmd.RegisterFlagCompletionFunc(staticIPFlagName, completion.AutocompleteNone)

```